src/AuditReader.php

```diff
@@ -216,7 +216,7 @@ public function find($className, $id, $revision, array $options = [])
                 $columnName = $idKeys[0];
             } elseif (isset($classMetadata->fieldMappings[$idField])) {
                 $columnName = self::getMappingColumnNameValue($classMetadata->fieldMappings[$idField]);
-            } elseif (isset($classMetadata->associationMappings[$idField]['joinColumns'])) {
+            } elseif (!$this->config->areAssociationsDisabled() && isset($classMetadata->associationMappings[$idField]['joinColumns'])) {
```

Comment:
If 
```
isset($classMetadata->associationMappings[$idField]['joinColumns'])
```
but the associations are disabled you now won't enter in this elseif, so you will go in the else
```
throw new \RuntimeException('column name not found for'.$idField);
```

- Is it an expected behavior to get an exception ?
- I would expect a better error message

So I think the logic should be
```
elseif (isset($classMetadata->associationMappings[$idField]['joinColumns'])) {
     if ($this->config->areAssociationsDisabled()) {
         // custom behavior or error message
     } else {
         $columnName = $classMetadata->associationMappings[$idField]['joinColumns'][0]['name'];
     }
} else {
     ...
}
```